### PR TITLE
Fix bug & Update build files

### PR DESCRIPTION
**Pull request by ksh8281**

### CODE REVIEW

**Comment by ksh8281** [comment]
fixes #1227

**Review by clover2123 — COMMENTED** [review]
LGTM with minor fixes

File: packaging/escargot.spec [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+CXXFLAGS+=' -Os '
+%endif
+
+%if "%{?enable_test}" == "1"
+CFLAGS+=' -DESCARGOT_ENABLE_TEST -DESCARGOT_ENABLE_TEMPORAL '
+CXXFLAGS+=' -DESCARGOT_ENABLE_TEST -DESCARGOT_ENABLE_TEMPORAL '
+%endif
+
+cmake CMakeLists.txt -H./ -Bbuild/out_tizen_%{rpm} -DLIBDIR=%{_libdir} -DINCLUDEDIR=%{_includedir} -DTIZEN_MAJOR_VERSION='%{tizen_version_major}' \
+-DESCARGOT_ARCH='%{tizen_arch}' -DESCARGOT_WASM='%{enable_wasm}' -DESCARGOT_DEBUGGER='%{enable_debugger}' -DESCARGOT_WASM='%{enable_wasm}' \
```
Comment (clover2123) [inline packaging/escargot.spec:260]: `-DESCARGOT_WASM` is set twice here

File: src/interpreter/ByteCodeInterpreter.cpp [hunk 2]
```diff
@@ -1760,8 +1760,23 @@ NEVER_INLINE void InterpreterSlowPath::resolveNameAddress(ExecutionState& state,
...
-            return;
+            // NOTE checking unscopables on hasBinding is impossible.
+            // because storeByNameWithAddress needs to call hasBinding also.
+            // If envRec.[[IsWithEnvironment]] is false, return true. skipped
+            // ObjectEnvironmentRecord is created only for with statement, so `IsWithEnvironment` is always true.
+            bool foundUnscopables = false;
+            if (env->record()->isObjectEnvironmentRecord()) {
+                ObjectPropertyName propertyName(code->m_name);
+                auto obj = env->record()->asObjectEnvironmentRecord()->bindingObject();
+                Value unscopables = env->record()->asObjectEnvironmentRecord()->bindingObject()->get(state, ObjectPropertyName(state.context()->vmInstance()->globalSymbols().unscopables)).value(state, obj);
```
Comment (clover2123) [inline src/interpreter/ByteCodeInterpreter.cpp:1771]: It seems like that `env->record()->asObjectEnvironmentRecord()->bindingObject()->get` can be simply replaced with `obj->get`

File: packaging/escargot.spec (hunk 1 above)
Comment (ksh8281) [reply packaging/escargot.spec:260]: Opps!

File: src/interpreter/ByteCodeInterpreter.cpp (hunk 2 above)
Comment (ksh8281) [reply src/interpreter/ByteCodeInterpreter.cpp:1771]: I see

**Review by clover2123 — APPROVED** [review]

**Pull request merged**